src/config.h

```diff
@@ -156,6 +156,9 @@ struct Config {
     // For example, to hide all files in a /CACHE/ folder, use ".*/CACHE/.*"
     std::vector<std::string> includeBlacklist;
     std::vector<std::string> includeWhitelist;
+
+    //
```

Comment:
Make sure to document all possible options here

Reply:
Done